smf/conch.xml

```diff
@@ -19,9 +19,9 @@
         </dependency>
 
 
-        <exec_method name='start' type='method' exec='/opt/conch/smf/conch' timeout_seconds='60'/>
+        <exec_method name='start' type='method' exec='/opt/conch/smf/conch start' timeout_seconds='60'/>
         <exec_method name='stop' type='method' exec=':kill' timeout_seconds='60'/>
```

Comment:
Use the `stop` via the script, which kills workers gracefully

Reply:
Done.